test/Sentry.Unity.Tests/IntegrationTests.cs

```diff
@@ -210,7 +210,7 @@ public IEnumerator DebugLogException_IsMarkedUnhandled()
     {
         yield return SetupSceneCoroutine("1_BugFarm");
 
-        var expectedMechanism = "\"mechanism\":{\"type\":\"Unity.LogException\",\"handled\":false}}]}";
+        var expectedMechanism = "\"mechanism\":{\"type\":\"Unity.LogException\",\"handled\":false";
```

Comment:
The `mechanism` now also contains the `,"data":{"HResult":"magicnumber"}`. So we're trimming the check because that's not what we're testing here.